src/Covenant/Prim.hs

```diff
@@ -279,3 +295,128 @@ data SixArgFunc
 instance Arbitrary SixArgFunc where
   {-# INLINEABLE arbitrary #-}
   arbitrary = elements [ChooseData, CaseData]
+
+-- | Maps one-argument functions to their input and output types.
+-- Returns a tuple of (arg type, result type).
+--
+-- @since 1.0.0
+typeOfOneArgFunc :: OneArgFunc -> (TyConstant, TyConstant)
+typeOfOneArgFunc = \case
+  LengthOfByteString -> (TyByteString, TyInteger)
+  Sha2_256 -> (TyByteString, TyByteString)
+  Sha3_256 -> (TyByteString, TyByteString)
+  Blake2b_256 -> (TyByteString, TyByteString)
+  EncodeUtf8 -> (TyString, TyByteString)
+  DecodeUtf8 -> (TyByteString, TyString)
+  FstPair -> (TyPair TyPlutusData TyPlutusData, TyPlutusData)
+  SndPair -> (TyPair TyPlutusData TyPlutusData, TyPlutusData)
+  HeadList -> (TyList TyPlutusData, TyPlutusData)
+  TailList -> (TyList TyPlutusData, TyList TyPlutusData)
```

Comment:
@kozross This PR doesn't include rank-1 polymorphism. I'll add that in a future PR.
For now, I'm using `TyPlutusData` as a placeholder for polymorphic types.